server_environment/serv_config.py

```diff
@@ -29,29 +30,37 @@
 
 from .system_info import get_server_environment
 
-from openerp.addons import server_environment_files
-_dir = os.path.dirname(server_environment_files.__file__)
+_logger = logging.getLogger(__name__)
 
 # Same dict as RawConfigParser._boolean_states
 _boolean_states = {'1': True, 'yes': True, 'true': True, 'on': True,
                    '0': False, 'no': False, 'false': False, 'off': False}
 
-if not system_base_config.get('running_env', False):
-    raise Exception(
-        "The parameter 'running_env' has not be set neither in base config "
-        "file option -c or in openerprc.\n"
-        "We strongly recommend against using the rc file but instead use an "
-        "explicit config file with this content:\n"
-        "[options]\nrunning_env = dev"
-    )
+try:
+    from openerp.addons import server_environment_files
 
-ck_path = os.path.join(_dir, system_base_config['running_env'])
+    _dir = os.path.dirname(server_environment_files.__file__)
 
-if not os.path.exists(ck_path):
-    raise Exception(
-        "Provided server environment does not exist, "
-        "please add a folder %s" % ck_path
-    )
+    if not system_base_config.get('running_env', False):
+        raise Exception(
+            "The parameter 'running_env' has not be set neither in base "
```

Comment:
Though I know you didn't write this exception, It could use a little sprucing up. Might I suggest:
```
            "The parameter 'running_env' is not set in the base config file, "
            "option -c or in openerprc.\n"
            "We strongly discourage the use of an rc file and instead "
            "recommend you explicitly add the following options into your "
            "config:\n[options]\nrunning_env = [dev|prod]"
```